atomic_reactor/plugins/exit_delete_from_registry.py

```diff
@@ -37,7 +39,22 @@ def __init__(self, tasker, workflow, registries):
         """
         super(DeleteFromRegistryPlugin, self).__init__(tasker, workflow)
 
-        self.registries = deepcopy(registries)
+        if check_variable(self.workflow, get_registries):
+            registries_cm = {}
+            all_registries = get_registries(self.workflow)
+            for registry in all_registries:
+                reguri = RegistryURI(registry.get('url'))
```

Comment:
Can we make `get_registries` return a list of `RegistryURI` objects? Maybe we need to enhance `RegistryURI` to also store `auth` information.